### PR TITLE
Add elastic APM config changes

### PR DESCRIPTION
**Pull request by minion96**
## JIRA Ticket Number

JIRA TICKET: ADO-5048

## Description of change
Add code for elastic APM integration

## Checklist:

- [ ] List of all ENV's needed
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

File: chatbot_ner/settings.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 from chatbot_ner.setup_sentry import setup_sentry
 
 BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+HAPTIK_ENV = os.environ.get('HAPTIK_ENV')
```
Comment (chiragjn) [inline chatbot_ner/settings.py:19]:
```suggestion
ENVIRONMENT = os.environ.get('ENVIRONMENT') or os.environ.get('HAPTIK_ENV')
```

File: chatbot_ner/settings.py [hunk 2]
```diff
@@ -34,6 +35,26 @@
 
 setup_sentry()
 
+# APM
+ELASTIC_APM_ENABLED = os.environ.get('ELASTIC_APM_ENABLED')
+ELASTIC_APM_ENABLED = True if ELASTIC_APM_ENABLED == 'True' and 'test' not in sys.argv else False
```
Comment (chiragjn) [inline chatbot_ner/settings.py:40]:
```suggestion
_elastic_apm_enabled = (os.environ.get('ELASTIC_APM_ENABLED') or '').strip().lower()
ELASTIC_APM_ENABLED = (_elastic_apm_enabled == 'true') and 'test' not in sys.argv
```

File: chatbot_ner/settings.py [hunk 3]
```diff
@@ -34,6 +35,26 @@
...
+ELASTIC_APM_SERVER_URL = os.environ.get('ELASTIC_APM_SERVER_URL')
+if ELASTIC_APM_ENABLED:
+    ELASTIC_APM = {
+        'DEBUG': DEBUG,
+        'SERVICE_NAME': 'chatbot_ner',
+        'SERVER_URL': ELASTIC_APM_SERVER_URL,
+        'SPAN_FRAMES_MIN_DURATION': '5ms',
+        'STACK_TRACE_LIMIT': 500,
+        'ENVIRONMENT': HAPTIK_ENV,
```
Comment (chiragjn) [inline chatbot_ner/settings.py:49]:
```suggestion
        'ENVIRONMENT': ENVIRONMENT,
```

File: chatbot_ner/settings.py [hunk 4]
```diff
@@ -59,6 +83,9 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+if ELASTIC_APM_ENABLED:
```
Comment (chiragjn) [inline chatbot_ner/settings.py:86]: Can we please move lines 38-56 here and club everything, something like,
```python
_elastic_apm_enabled = (os.environ.get('ELASTIC_APM_ENABLED') or '').strip().lower()
ELASTIC_APM_ENABLED = ...
ELASTIC_APM_SERVER_URL = ...
if ELASTIC_APM_ENABLED:
    ELASTIC_APM = ...
    INSTALLED_APPS.append('elasticapm.contrib.django')
    MIDDLEWARE.append('elasticapm.contrib.django.middleware.TracingMiddleware')
```

**Review by chiragjn — CHANGES_REQUESTED** [review]
requesting minor changes

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

File: chatbot_ner/settings.py (hunk 4 above)
Comment (minion96) [reply chatbot_ner/settings.py:67]: This is done

**Review by chiragjn — APPROVED** [review]

**Comment by minion96** [comment]
Retest this please

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

**Pull request merged**